### ISSUE TITLE
Book 3.5.3: pScatter(theta_o < 0) = 0 is incorrect

### ISSUE THREAD

**Issue by gau-nernst** [RayTracing/raytracing.github.io#1331]
In chapter 5.3. The scattering PDF, when we integrate over the hemisphere, it is written that $pScatter(\theta_o<0) = 0$. I believe the correct one should be $pScatter(\pi/2 < \theta_o < \pi) = 0$ (note that $\theta_o \in [0, \pi]$). The equation below that is correct. Since $pScatter(\pi/2 < \theta_o < \pi) = 0$, we only need to integrate $\theta_o$ from 0 to $\pi/2$.

Furthermore, to help with understanding, should we add the variable name in the integral sign? i.e. replace $\int_0^{2\pi} \int_0^{\pi/2}$ with $\int_{\phi=0}^{2\pi} \int_{\theta=0}^{\pi/2}$.

**Comment by hollasch** [comment]
Good grief, that took me way too long to parse out. I was stuck thinking of $\phi$ as the vertical angle, and $\theta$ as the angle around the vertical axis, which is apparently my standard spherical coordinate notation.

Instead, $\theta$ is the angle between the outgoing direction vector and the surface normal vector, where an angle of $0$ means an outgoing direction vector along the surface normal, and $\pi$ means an outgoing direction vector directly opposite the surface normal.

I am still at a loss to understand what the $o$ subscript means ($\phi_o$ and $\theta_o$).

Anyway, long story short, your suggestion looks good.
